db/seeds.rb

```diff
@@ -18,6 +18,7 @@
     beginning_at: date,
     end_at: date + 2.days,
     category: categories.sample,
-    description: Faker::HipsterIpsum.words(250).join(' ').capitalize
+    description: Faker::HipsterIpsum.words(250).join(' ').capitalize,
+    user_id: rand(1..5)
```

Comment:
Got it. Yeah it's possible but also wierd. I will log into application frist time and I will have events already which I didn't added. We, developers will know what's going on but normal user not. Please create a dummy users in the seeds and assign events to them :)